package.json

```diff
@@ -437,7 +437,7 @@
     "chai-string": "^1.5.0",
     "concurrently": "^9.0.1",
     "copyfiles": "^2.4.1",
-    "mariadb": "^3.3.1",
+    "mariadb": "^3.3.2",
```

Comment:
Shouldn't this be a seperate pr?

Reply:
Theoretically yes, but the same errors in the action were present in both PRs. That's why I decided to increase the mariadb version here as well, so that both errors were fixed at once.